substrait_consumer/functional/queries/sql/relations/project_relations.py

```diff
@@ -0,0 +1,72 @@
+from substrait_consumer.producers.duckdb_producer import DuckDBProducer
+from substrait_consumer.producers.datafusion_producer import DataFusionProducer
+
+PROJECT_RELATIONS = {
+    "project_single_col": (
+        """
+        SELECT L_DISCOUNT
+        FROM '{}'
+        ORDER BY L_DISCOUNT
```

Comment:
Is it possible to avoid the order by?  That ends up making the test less pure which makes it harder to identify issues and read the output.

Reply:
the order by is being used to keep consistent results.  do you think it would be fine here to do some type of sorting and comparison after the substrait results output and comparing based on that?